### PR TITLE
Update 'SPIRV-Headers' to v1.5.4.raytracing.fixed

### PR DESCRIPTION
**Pull request by zbai-sc**
* I've followed [this guide](https://hunter.readthedocs.io/en/latest/creating-new/update.html)
  step by step carefully. **Yes**

---

### CODE REVIEW

File: cmake/projects/SPIRV-Headers/hunter.cmake [hunk 1]
```diff
@@ -43,6 +43,28 @@ hunter_add_version(
...
+    https://github.com/KhronosGroup/SPIRV-Headers/archive/1.5.4.tar.gz
+    SHA1
+    24e517cc55339b847ccb7d3b1d59f73b7df775a1
+)
+
+hunter_add_version(
+    PACKAGE_NAME
+    SPIRV-Headers
+    VERSION
+    1.5.4.raytracing.fixed
```
Comment (rbsheth) [inline cmake/projects/SPIRV-Headers/hunter.cmake:61]: Can you set the default config to use this latest version?

Comment (zbai-sc) [reply cmake/projects/SPIRV-Headers/hunter.cmake:61]: I'd suggest to wait util we bump up SPIRV-Tools to v2020.6. SPIRV-Tools are really picky about SPIRV-Headers version and their versions needs to be matched exactly.

Comment (rbsheth) [reply cmake/projects/SPIRV-Headers/hunter.cmake:61]: We still need to test this version, so let's update the default, check CI, and then downgrade if necessary.

Comment (zbai-sc) [reply cmake/projects/SPIRV-Headers/hunter.cmake:61]: done

**Review by rbsheth — APPROVED** [review]

**Comment by rbsheth** [comment]
Released: https://github.com/cpp-pm/hunter/releases/tag/v0.23.291

**Pull request merged**